plugins/cloudbuild/src/components/useProjectName.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useAsync } from 'react-use';
+import { Entity } from '@backstage/catalog-model';
+
+export const CLOUDBUILD_ANNOTATION = 'google.com/cloudbuild';
```

Comment:
a bit unclear what the format of this one is - either we could call it `google.com/cloudbuild-project-slug` similar to the other [well known annotations](https://backstage.io/docs/features/software-catalog/well-known-annotations) if it's just a project, or a generic `google.com/project-slug` if it is commonly the case that all features are keyed off of the same thing.